electrum/plugins/revealer/qt.py

```diff
@@ -312,14 +324,19 @@ def make_rawnoise(self, create_revealer=False):
             self.noise_seed = int(self.noise_seed, 16)
             self.hex_noise = self.version + str(format(self.noise_seed, '02x'))
 
-
         self.code_id = self.code_hashid(self.hex_noise)
         self.hex_noise = ' '.join(self.hex_noise[i:i+4] for i in range(0,len(self.hex_noise),4))
-        random.seed(self.noise_seed)
+        entropy = binascii.unhexlify(str(format(self.noise_seed, '02x')))
```

Comment:
This will fail sometimes.
`format(1000, '02x') == '3e8'` << has odd length, and binascii.unhexlify will raise
please fix for the other usages too, there are at least two just above